### ISSUE TITLE
disable scrolling of visualization

### ISSUE THREAD

**Issue by herbertblb** [erictapen/mapping-migration#57]
Right now, it is possible to scroll vertically and horizontally although there is nothing to see. Is it possible to disable this? Only the introduction should be scrollable, the rest should be static.

**Comment by erictapen** [comment]
Yeah, that's a problem I'm still unsure how to tackle. Chances are that I won't find a good solution for this...

**Comment by herbertblb** [comment]
no problem!

**Comment by erictapen** [comment]
Maybe I finally fixed this in e89ed39c9bf5ffa79b6cf8221bd8155969fe74cd?
